openpype/hosts/maya/plugins/publish/collect_pointcache.py

```diff
@@ -45,15 +45,26 @@ def process(self, instance):
 
         # Collect user defined attributes.
         if not instance.data.get("includeUserDefinedAttributes", False):
-            return
+            user_defined_attributes = set()
```

Comment:
The logic for the backwards compatibility attribute for this should trigger above this - now this doesn't collected for the backwards compatibility attribute.